### PR TITLE
DFR-2889 workaround

### PR DESCRIPTION
**Pull request by jthmcts**
### Jira link (if applicable)
https://tools.hmcts.net/jira/browse/DFR-2889

### Change description ###
DFR-2889 Enable HighLevelDataSetup in preview.
Includes workaround for https://tools.hmcts.net/jira/browse/CCD-5362

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -68,17 +75,13 @@ dependencies {
...
-    testImplementation group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.21.22'
+    testImplementation group: 'org.assertj', name: 'assertj-core', version: '3.26.0'
+    testImplementation group: 'org.slf4j', name: 'slf4j-api', version: '2.0.13'
 
     testRuntimeOnly group: 'org.glassfish', name: 'jakarta.el', version: '4.0.2'
-    testImplementation group: 'org.projectlombok', name: 'lombok', version: '1.18.34'
+    testCompileOnly group: 'org.projectlombok', name: 'lombok', version: '1.18.34'
     testAnnotationProcessor group: 'org.projectlombok', name: 'lombok', version: '1.18.34'
-}
+}
```
Comment (ptrelease) [inline build.gradle:87]: Missing new line?

Comment (jthmcts) [reply build.gradle]: New line added.

File: src/test/java/finrem/HighLevelDataSetupApp.java [hunk 2]
```diff
@@ -1,39 +1,114 @@
...
+     */
+    @Override
+    protected void importDefinitionsAt(String definitionsPath) {
+        List<String> definitionFileResources = getAllDefinitionFilesToLoadAt(definitionsPath);
+        logger.info("{} definition files will be uploaded to '{}' on {}.", definitionFileResources.size(),
+                BeftaMain.getConfig().getDefinitionStoreUrl(), getDataSetupEnvironment());
+        String message = "Couldn't import {} - Exception: {}.\n\n";
+        try {
+            for (String fileName : definitionFileResources) {
+                try {
```
Comment (hamzahtahirhmcts) [inline src/test/java/finrem/HighLevelDataSetupApp.java:99]: Should we move this second try block into a separate method? We have nested try catch block so perhaps we can move this out into a method like `importSingleDefinition(String fileName)` and then move the `message` variable into that method too 🤔

Comment (jthmcts) [reply src/test/java/finrem/HighLevelDataSetupApp.java]: This code has been copied from https://github.com/hmcts/befta-fw/blob/b030e3e954bcb5920854022f8d3f8bb59f2ee9de/src/main/java/uk/gov/hmcts/befta/dse/ccd/DataLoaderToDefinitionStore.java#L328

Essentially I want to do exactly the same as the parent class but not throw any RuntimeException in preview so ideally I want to minimise changes but I accept your comment (perhaps you could create a PR on their repo too 😉 )

**Review by hamzahtahirhmcts — APPROVED** [review]

**Review by so99y — APPROVED** [review]
LGTM

**Review by ptrelease — APPROVED** [review]

**Pull request merged**